### PR TITLE
Bridge 22.7

### PR DESCRIPTION
**Pull request by madeofstown**
I added the bridge mode file because i hadn't seen one that worked with OPNsense 22.7 yet.

### CODE REVIEW

File: bin/opnatt-bridge.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+
+ONT_IF='igb0'
+RG_IF='igb1'
+RG_ETHER_ADDR='xx:xx:xx:xx:xx:xx'
+LOG=/var/log/pfatt.log
```
Comment (owenthewizard) [inline bin/opnatt-bridge.sh:7]: These `pfatt` references should be changed to `opnatt`.

**Comment by owenthewizard** [comment]
Should this not be merged upstream into pfatt instead? My fork is intended to be specific to the supplicant method, which, it seems to me, does not get proper attention as part of pfatt.

**Comment by madeofstown** [comment]
I assumed that your fork of pfatt and renaming to opnatt was an indication of a specialized version for OPNSense. Seems that it would be better to have a both bypass methods available in the same repo ala pfatt. Also, the original pfatt repo is not maintained or updated.

**Comment by owenthewizard** [comment]
Closing this as I don't intend to support both supplicant and bridge bypasses.

**Pull request closed without merging**